### PR TITLE
Improve Cacheability of Commercial Requests

### PR DESCRIPTION
**Pull request by kelvin-chappell**

### CODE REVIEW

**Comment by samdesborough** [comment]
:+1:

File: static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js [hunk 1]
```diff
@@ -47,6 +47,8 @@ define([
                     }).join('&');
                 }).join('&');
         },
+        /*
+        To reinstate when find out where required
```
Comment (johnduffell) [inline static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js]: awesome comment! ;)

Comment (kelvin-chappell) [inline static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js]: I detect some sarcasm

Comment (johnduffell) [inline static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js]: actually I really just thought it was funny! but actually on reflection it is a good comment cos it explains what you were *thinking* when you made the change

**Pull request merged**